### PR TITLE
docs: editorial pass on conditions_and_events.md

### PR DESCRIPTION
**Pull request by zbwright**
[skip ci]

### CODE REVIEW

File: doc/user/conditions_and_events.md [hunk 1]
```diff
@@ -1,18 +1,19 @@
-# Status Conditions and Events
+# Status Events and Conditions
```
Comment (fanminshi) [inline doc/user/conditions_and_events.md]: is there a reason for switch the order ?

Comment (hongchaodeng) [reply doc/user/conditions_and_events.md]: I think it is because we talked events first and conditions second in this doc.
So this is fine to me.

**Comment by hongchaodeng** [comment]
lgtm

File: doc/user/conditions_and_events.md (hunk 1 above)
Comment (fanminshi) [reply doc/user/conditions_and_events.md]: kk. sounds good.

**Comment by fanminshi** [comment]
lgtm.

**Pull request merged**